cirq/google/optimizers/convert_to_sycamore_gates.py

```diff
@@ -43,14 +45,19 @@ class ConvertToSycamoreGates(circuits.PointOptimizer):
         Otherwise raises a TypeError.
     """
 
-    def __init__(self, ignore_failures=False) -> None:
+    def __init__(self,
+                 tabulation: Optional[GateTabulation] = None,
+                 ignore_failures=False) -> None:
         """
         Args:
+            tabulation: If set, a tabulation for the Sycamore gate to use for
+                decomposing gates.
```

Comment:
re: @dkafri 's suggestion. I'd expand on this docstring more

 1. tabulation only used for matrix gate
 2. otherwise you use another scheme

Reply:
3. how do I get a tabulation

Reply:
Done.